packages/components/src/Icon/Icon.component.js

```diff
@@ -36,12 +36,27 @@ export const TRANSFORMS = Object.keys(FA_TRANSFORMS);
  * @example
 <Icon name="fa-bars"></Icon>
  */
-function Icon({ className, name, title, transform, onClick }) {
+function Icon({ className, name, title, transform, onClick, src }) {
 	const accessibility = {
 		focusable: 'false', // IE11
 		'aria-hidden': 'true',
 		title: title || null,
 	};
+	if (src) {
+		const classNames = classnames(
+			theme['tc-icon'],
+			'tc-icon',
+			className
+		);
+		if (onClick) {
+			return (
+				<a xlinkHref="#" onClick={onClick} className={classnames('tc-svg-anchor', theme.link)}>
```

Comment:
xlinkHref is deprecated + for accessibility issue i think you should use a button element @frassinier